### PR TITLE
Small improvements and security fixes

### PR DESCRIPTION
**Pull request by pbanaszkiewicz**
This PR tries to address 1 CodeQL analysis issue, cleans up CodeQL workflow and removes unnecessary files (editor config and travisCI config).

### CODE REVIEW

**Comment by pbanaszkiewicz** [comment]
I need to figure out a way to "cheat" the code scanning results from CodeQL.

**Comment by pbanaszkiewicz** [comment]
This now fixes 1 alert from CodeQL.

File: amy/workshops/views.py [hunk 1]
```diff
@@ -1293,7 +1293,7 @@ def event_import(request):
         return HttpResponseBadRequest("Network connection error.")
 
     except WrongWorkshopURL as e:
-        return HttpResponseBadRequest(str(e))
+        return HttpResponseBadRequest(f"URL error: {e.msg}")
```
Comment (lauryndbrown) [inline amy/workshops/views.py:1296]: Could you tell me why e.msg is necessary?

File: amy/workshops/util.py [hunk 2]
```diff
@@ -681,14 +681,15 @@ def fetch_workshop_metadata(event_url, timeout=5):
...
+class WrongWorkshopURL(Exception):
     """Raised when we fall back to reading metadata from event's YAML front
     matter, which requires a link to GitHub raw hosted file, but we can't get
-    that link because provided URL doesn't match Event.WEBSITE_REGEX
+    that link because provided URL doesn't match Event.WEBSITE_REGEX or Event.REPO_REGEX
     (see `generate_url_to_event_index` below)."""
 
-    def __str__(self):
-        return "Event's URL doesn't match Github website or repo format."
+    def __init__(self, msg: str):
```
Comment (lauryndbrown) [inline amy/workshops/util.py:690]: I believe what you're hoping for is to just show the message of the error. This is achieved without altering the class:
```py
>>> class MyException(Exception):
...    pass
... 
>>> error = MyException("Hi")
>>> str(error)
'Hi'
```

**Review by lauryndbrown — APPROVED** [review]
I don't think those issues are blocking. It looks like codeql is working. :)

File: amy/workshops/util.py (hunk 2 above)
Comment (lauryndbrown) [reply amy/workshops/util.py:690]: Oh nevermind me then... looks like this was what you were trying to fix. Hmmm....

File: amy/workshops/views.py (hunk 1 above)
Comment (lauryndbrown) [inline amy/workshops/views.py:1296]: Could we leave a note here in the comments mentioning why this was done?

Comment (pbanaszkiewicz) [reply amy/workshops/views.py:1296]: I added the comment to the `WrongWorkshopURL` class.

**Pull request merged**